live_site.py

```diff
@@ -0,0 +1,433 @@
+import os
+import json
+import click
+import requests
+import colorama
+import digitalocean
+from colorama import Fore
+from colorama import init
+from pssh.clients import SSHClient
+
+
+if os.getenv('DIGITAL_OCEAN_AUTH_TOKEN'):
+	api_token = os.getenv('DIGITAL_OCEAN_AUTH_TOKEN')
+else:
+	api_token = input ('Enter your API token: ')
+
+
+if os.getenv('SSH_KEY'):
+	ssh_key = os.getenv('SSH_KEY')
+else:
+	ssh_key = input (' Enter your SSH Key path: ')
+
+# The api_url_base variable is the string that starts off every URL in the DigitalOcean API.
+api_url_base = 'https://api.digitalocean.com/v2/'
+# We need to set up the HTTP request headers the way the API docs describe.
+# Add these lines to the file to set up a dictionary containing your request headers:
+headers = {'Content-Type': 'application/json',
+		'Authorization': 'Bearer {0}'.format(api_token)}
+
+
+
+def get_all_droplets():
+	api_url = '{0}droplets'.format(api_url_base)
+	response = requests.get(api_url, headers=headers)
+
+	if response.status_code == 200:
+		return json.loads(response.content.decode('utf-8'))
+	else:
+		return None
+
+def del_droplet(dropletId):
+
+	droplet = digitalocean.Droplet(token=api_token,id=dropletId)
+
+	destroyed = droplet.destroy()
+	click.secho(">>> Success! Your droplet has been deleted.", fg='bright_green')
+	
+def del_domain(domain_name):
+
+	domain = digitalocean.Domain(token=api_token, name=domain_name)
```

Comment:
What happens if the domain name doesn't exist?

I think it will probably still work, but I'm curious what exactly happens